pom.xml

```diff
@@ -388,6 +388,12 @@
                             <artifactId>snakeyaml</artifactId>
                             <version>1.33</version>
                         </exlude>
+
+                        <exlude>
+                            <groupId>com.google.guava</groupId>
```

Comment:
We should really raise a ticket to fix this and link it here like the ones above.